Description: Drop a test-suite experiment committed by mistake
 This was failing on armhf because Debian GCC now enables PIE by default.
Author: Xavier Leroy <xavier.leroy@inria.fr>
Bug: https://caml.inria.fr/mantis/view.php?id=7642
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=837359#80
---
diff --git a/testsuite/tests/asmcomp/main.ml b/testsuite/tests/asmcomp/main.ml
index 284f7fb..6457ad2 100644
--- a/testsuite/tests/asmcomp/main.ml
+++ b/testsuite/tests/asmcomp/main.ml
@@ -6,7 +6,6 @@ let compile_file filename =
     let out_name = Filename.chop_extension filename ^ ".s" in
     Emitaux.output_channel := open_out out_name
   end; (* otherwise, stdout *)
-  Clflags.dlcode := false;
   Compilenv.reset ~source_provenance:(Timings.File filename) "test";
   Emit.begin_assembly();
   let ic = open_in filename in
